autoscheduler/frontend/src/components/NavBar.tsx

```diff
@@ -39,29 +51,34 @@ const NavBar: React.SFC = () => {
         color="primary"
       >
         <Toolbar>
-          <div className={classes.title}>
```

Comment:
GitHub really mangled these changes, I compared it locally and you just added 5 lines but it looks like you completely reorganized the file here lmao